src/dashboard/src/main/models.py

```diff
@@ -481,18 +481,28 @@ def __unicode__(self):
         }
 
     @classmethod
-    def create_many(cls, dir_paths_uuids, unit_mdl, unit_type='transfer'):
+    def create_many(cls, dir_paths_uuids, unit_mdl, unit_type='transfer',
+                    original_location=None):
         """Create ``Directory`` models to encode the relationship between each
-        directory path/UUID pair in ``dir_paths_uuids`` and the ``Transfer`` model
-        that the directories are a part of.
+        directory path/UUID entry in ``dir_paths_uuids`` and the ``Transfer``
+        model that the directories are a part of. ``dir_paths_uuids`` is
+        expected to be a dict instance where the ``originallocation`` field is
+        optional. The ``originallocation`` but can be set according to the
+        requirements of the PREMIS record that eventually needs to be created
+        using this model.
         """
-        unit_type = {'transfer': 'transfer'}.get(unit_type, 'sip')
-        return cls.objects.bulk_create([
-            cls(**{'uuid': dir_uuid,
-                   unit_type: unit_mdl,
-                   'originallocation': dir_path,
-                   'currentlocation': dir_path})
-            for dir_path, dir_uuid in dir_paths_uuids])
+        paths = []
+        for dir_ in dir_paths_uuids:
+            orig_path = None
```

Comment:
Not needed?

Reply:
This function can create many new directory entries. If `originalLocation` is passed via the iterator that can be passed to this function then it overrides the `currentLocation` so I believe it is needed. Is there a reason you believe it shouldn't be here? 

The named argument `original_location=None` will be removed, so maybe this was the problem?

Reply:
> Is there a reason you believe it shouldn't be here?

I only thought it was not needed because `orig_path` is assigned a new value in L499 anyways.

Reply:
No worries :)